user-org-actor/src/main/java/org/sunbird/factory/ServiceFactory.java

```diff
@@ -0,0 +1,18 @@
+package org.sunbird.factory;
+
+import org.sunbird.ServiceImplType;
+import org.sunbird.user.service.UserServiceImpl;
+
+/**
+ * This class is created for keeping all service's object creation.
+ */
+public class ServiceFactory  {
+
+    public static Object getService(String serviceName) {
+        Object serviceObj = null;
+        if(serviceName.equals(ServiceImplType.USER.getServiceType())) {
```

Comment:
DO reverse check to avoid NPE.  
 serviceName.equals(ServiceImplType.USER.getServiceType())  to  ServiceImplType.USER.getServiceType().equals(serviceName)